SEED-platform/seed
# 3301 add ghg units to org model

#### Any background context you want to provide?
SEED columns with data types of 'area' and 'eui' were appended with a unit in the inventory detail and inventory list, e.g. "Gross Floor Area (ft²)". GHG and GHG Intensity columns were displayed without units, despite having hardcoded unit values on the PropertyState model.

#### What's this PR do?
- Updates the organization model to recognize `ghg` and `ghg_intensity` units 
- Allows users to select GHG and intensity units on the mapping page. 
- Adds GHG and intensity units to column headers on the inventory list and detail

Note: Due to Pints pretty formatting the following units are converted for printing `kgCO2e --> kg` and `MtCO2e --> t` 

#### How should this be manually tested?
- Upload a property file with GHG and GHG intensity columns
- confirm the mapping screen has a dropdown of the correct unit type (currently only one available)
- confirm GHG and intensity column headers have correct units appended on the inventory list and display

#### What are the relevant tickets?
#3301 

#### Screenshots (if appropriate)

![Screen Shot 2022-07-21 at 3 58 34 PM](https://user-images.githubusercontent.com/58446472/180322483-41e16c68-2e7c-4b6c-ac1b-96d7be1402e3.png)
![Screen Shot 2022-07-21 at 3 56 37 PM](https://user-images.githubusercontent.com/58446472/180322487-79142514-917b-477b-8fb3-8842396d42c2.png)
![Screen Shot 2022-07-21 at 3 56 10 PM](https://user-images.githubusercontent.com/58446472/180322489-a69f358c-5f50-43e0-88d7-5e10ae1f613f.png)
